File: javascript/Count of positives sum of negatives.js
function countPositivesSumNegatives(input) {
  
    if(!input|| input.length ===0 ){
      return []
    }
    
    let posCount = 0;
    let sumNeg = 0;
    
    for(let i = 0; i < input.length; i++){
      if(input[i] < 0){
        sumNeg += input[i];
      }
      if(input[i]>0){
        posCount += 1;
      }
    }
    return[posCount, sumNeg]
  }

File: python/Count of positives sum of negatives.py
def count_positives_sum_negatives(arr):
    if not arr:
        return []
    
    posCount = 0;
    negSum = 0;
    
    for i in arr:
        if i < 0:
            negSum += i
        elif i > 0:
            posCount += 1
    return [posCount,negSum] 